packages/sdk/src/referral.ts

```diff
@@ -290,6 +352,67 @@ export class ReferralProvider {
     };
   }
 
+  public async getReferralTokenAccountsWithStrategyV2(
+    referralAccountAddress: string,
+    strategy: GetReferralTokenAccountsStrategy = {
```

Comment:
we don't have to support strategy here. since we can actually just retrieve all accounts and let them claim.